backend/src/main/java/co/inventorsoft/scripty/repository/RoleRepository.java

```diff
@@ -0,0 +1,10 @@
+package co.inventorsoft.scripty.repository;
+import co.inventorsoft.scripty.entity.Role;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.Optional;
+
+public interface RoleRepository extends JpaRepository<Role, Long> {
+    Optional<Role> findById(Long id);
```

Comment:
You don;t need to define this method, cause it's already present in the commons https://docs.spring.io/spring-data/commons/docs/current/api/org/springframework/data/repository/CrudRepository.html#findById-ID-